client/index.jsx

```diff
@@ -27,37 +34,70 @@ class App extends React.Component {
         .then((data) => {
           this.setState({
             reviews: data.data,
+            filteredReviews: data.data,
           });
         })
         .catch((err) => console.error(err));
     }
   }
 
+  filterReviews(key, value) {
+    const { reviews } = this.state;
+    const { filterToggle } = this.state;
+    if (filterToggle.starRaiting === value) {
```

Comment:
Raiting => Rating

Reply:
might want to find all for Raiting. There's at least one other instance.

Reply:
This will be the death of me 😅